CHANGELOG.md

```diff
@@ -4,10 +4,23 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.1.0 (2021-02-22)
+
+This release adds major new features since the 2.0.2 release.
+We deem it moderate priority for upgrading.
+
+This release adds the long-awaited support for PostgreSQL 13 to TimescaleDB.
```

Comment:
Perhaps we should say something about minimum supported PG version (13.2), and mention security issues discovered in previous versions?

Reply:
I agree. @svenklemm could you please complete with that information.?

Reply:
done